php/Terminus/Models/Collections/CodeLogs.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Terminus\Models\Collections;
+
+class CodeLogs extends TerminusCollection {
```

Comment:
Sorry, could we name these "Commits"? I know that's not what the API path is, but they're actually commits with a git hash and parents and the whole shebang.